website/server/controllers/api-v3/chat.js

```diff
@@ -300,6 +300,52 @@ api.clearChatFlags = {
       {$set: {'chat.$.flagCount': message.flagCount}}
     );
 
+    let adminEmailContent;
+    if (user.auth.local) {
```

Comment:
here to get the email addresses could you use the `email.getUserInfo(user, ['email'])` function from https://github.com/HabitRPG/habitrpg/blob/develop/website/server/libs/api-v3/email.js#L46? It would be great if you could change it here too https://github.com/HabitRPG/habitrpg/blob/develop/website/server/controllers/api-v3/chat.js#L212